e2e-tests/src/test/rewards.rs

```diff
@@ -109,3 +120,74 @@ pub fn disable_node(config: &Config) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub fn force_new_era(config: &Config) -> anyhow::Result<()> {
+    const MAX_DIFFERENCE: f64 = 0.05;
+
+    let node = &config.node;
+    let accounts = get_validators_keys(config);
+    let sender = accounts.first().expect("Using default accounts").to_owned();
+    let connection = SignedConnection::new(node, sender);
+
+    let sudo = get_sudo_key(config);
+    let root_connection = RootConnection::new(node, sudo);
+
+    let reserved_members: Vec<_> = get_reserved_members(config)
+        .iter()
+        .map(account_from_keypair)
+        .collect();
+    let non_reserved_members: Vec<_> = get_non_reserved_members(config)
+        .iter()
+        .map(account_from_keypair)
+        .collect();
+
+    wait_for_full_era_completion(&connection)?;
+
+    let start_era = get_current_era(&connection);
+    let start_session = get_current_session(&connection);
+    info!("Start | era: {}, session: {}", start_era, start_session);
+
+    staking_force_new_era(&root_connection, XtStatus::Finalized);
+
+    wait_for_session(&connection, start_session + 2)?;
+    let current_era = get_current_era(&connection);
+    let current_session = get_current_session(&connection);
+    info!(
+        "After ForceNewEra | era: {}, session: {}",
+        current_era, current_session
+    );
+
+    // Once a new era is forced in session k, the new era does not come into effect until session
+    // k + 2; we test points:
+    // 1) immediately following the call in session k,
+    // 2) in the interim session k + 1,
+    // 3) in session k + 2, the first session of the new era.
+    for idx in 0..3 {
+        let session_to_check = start_session + idx;
+        let era_to_check = start_era + idx / 2;
+
+        info!(
+            "Testing points | era: {}, session: {}",
+            era_to_check, session_to_check
+        );
+
+        let non_reserved_for_session =
+            get_non_reserved_members_for_session(config, session_to_check);
+        let non_reserved_bench =
+            get_bench_members(non_reserved_members.clone(), &non_reserved_for_session);
+
+        check_points(
+            &connection,
+            session_to_check,
+            era_to_check,
+            reserved_members
```

Comment:
Nit: I think I would extract it into a separate `let` binding just for the sake of naming it (feel free to ignore this :))

Reply:
Good call. Made me think some more about the naming convention here and I included this along with minuscule changes in other variable names.